AFQ/definitions/utils.py

```diff
@@ -72,8 +72,8 @@ def _arglist_to_string(args, get_attr=None):
 def name_from_path(path):
     file_name = op.basename(path)  # get file name
     file_name = drop_extension(file_name)  # remove extension
-    if "-" in file_name:
-        file_name = file_name.split("-")[-1]  # get suffix if exists
+    if "_" in file_name:
```

Comment:
Are we even using this function?

Reply:
ParticipantAFQ uses this function to generate names for scalars that the user inputs (such as a t1 image from another pipeline). I should make it so the user can also define a name, and we call this function only if a name is not provided.